htdocs/web_portal/views/site/view_site.php

```diff
@@ -38,7 +38,7 @@
         <?php if ($params['ShowEdit']): ?>
         <div style="float: right; margin-left: 2em;">
```

Comment:
> > Why do in two of the files, the trash image gets called first, but in the other files the pencil gets called first?
> 
> 🤷 Probably not a good reason.
> 
> I'm further confused by view_sites.php where trash is defined first but appears to the right of the edit button, i.e. https://gocdb-preprod.egi.eu/portal/index.php?Page_Type=Site&id=335

I don't see either item at that link, I guess because i'm not an admin. But I can see it at sites on my instance.
I imagine because of this line, where pencil is given a margin-left

Reply:
Actually scratch that I think, thats just giving the whole thing a margin from the edge of the page

Reply:
ah yes, they are in seperate divs - one alligning left and one alligning right.

If you want to address the inconsitency, please do open an issue and tackle it 😄

Reply:
actually so has trash in that file - in the other files only pencil gets the margin I think